reddit/templates/__layout/sidebar.html

```diff
@@ -28,8 +28,8 @@ <h4 class="side-bar--sub-title">{{ raw_post_count }} Reviews</h4>
 
       <!-- ## link to post comes here!! -->
       <a href="#0">
```

Comment:
Hi @NkosanaKG we still need to insert a link here so that the user can click through to the post itself.